README.adoc

```diff
@@ -8,42 +8,430 @@ The time for the full course is around three to four days.
+Pricing
+~~~~~~~
+
+Ferrous Systems usually charges per trainer day. That allows for easy
+calculation on one-off trainings and a fair and easy discussion about
+the structure of the course. Other agreements are possible. For example,
+that allows clients to quickly change the setup of the group, e.g. by
+adding a person, without changing the contractual setup.
+
+We offer training on any Rust subject and will create material that we
+deem resuable at no cost for the client.
+
+We also offer training directly on clients products, against a fee to
+produce the material.
+
+Group setup
+~~~~~~~~~~~
+
+Mixed backgrounds allow facilitating discussions within a group about
+approaches and techniques. A similar experience level allows the trainer
+to form better assumptions about what is already understood and what
+not.
+
+Ferrous Systems recommends a trainer per 6 attendees for full trainer
+bandwidth. If possible, we recommend opting for groups of at least 12,
+to efficiently allow for 2 trainers.
+
+We have experiences with both groups of mixed and uniform backgrounds
+and mixed skill levels. We generally recommend groups of mixed
+backgrounds and similar experience levels.
+
+Training material
+~~~~~~~~~~~~~~~~~
+
+Ferrous Systems offers their training material under MIT/Apache-2.0
+license for source code and Creative Commons (CC-BY-SA) for textual
+content.
+
+Course material specifically constructed for you will be proprietary to
+you.
+
+Trainer staff
+~~~~~~~~~~~~~
+
+The Ferrous Systems trainer staff consists of multiple Rust project
+maintainers and domain experts, active in multiple working groups. This
+makes sure our trainers are always up to speed on new developments and
+can give current and authoritive answers.
+
+Among those are:
+
+* Florian Gilcher: Training Rust since 2015. Over 30 trainings, most for
+large enterprises. Rust core team member and community team manager.
+Broad knowledge about all aspects of Rust, with special knowledge on
+network and async/concurrent programming.
+* James Munns: Training Rust since 2018. Focus on general Rust and
+small/embedded systems. Member of the Embedded Working Group Core Team.
+Great knowledge of integrating Rust with C codebases.
+* Aleksey Kladov: Training Rust since 2018 in academic and enterprise
+setting. Member of the Rust tools team. Author of the rust-analyzer IDE.
+Very broad and exact knowledge of the language and its tooling,
+especially focused on debugging.
+* Jonas Schievek: Training Rust since 2019. Focus on general Rust and
+multiple architectures. Contributor to the Rust compiler. Great
+knowledge of compilation details and performance concerns.
+* Tanks Transfeld: Training Rust since 2018. Focus on general Rust and
+Rust for beginners. Part of several Rust learning projects. All Rust
+subjects, from async to bare metal. Great knowledge around beginners
+concerns and how to get started.
+
+Ferrous Systems can also collaborate with local trainers in Europe, US,
```

Comment:
We can probably also throw Canada on the list, or generalize to "North America".

Reply:
Added